microsoft-edge/webview2/release-notes.md

```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
 
 The following sections cover either a Release SDK package (1.0.####.##) or a Prerelease SDK package (1.0.####-prerelease).
 
+<!-- ====================================================================== -->
+## 1.0.1774.30
+
+Release Date: May 8, 2023
+
+[NuGet package for WebView2 SDK 1.0.1774.30](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1774.30 or higher.

```

Comment:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 113.0.1774.30 or higher.
```
@nishitha-burman @novac42 - change from 112 to 113?

Reply:
113 is correct.